### DISCUSSION TITLE
Latest actions

### DISCUSSION THREAD

**Discussion by JosephJilek** [Ralim/IronOS#1085] [Q&A] [answered]
Do the latest actions/cutting edge code include all previous actions?

**Comment by JosephJilek** [comment]
Such as, if I download [changed power animation (#1082) CI #1049], would it include [change increment and values of brightness settings (#1083) CI #1047]?

**Reply by Ralim** [reply]
To add to the answer, it depends on where they are based on.
If the PR is forked out from the main branch before the later changes then they _may_ not be included.
I normally try to "sync" the PR's though to minimise this.

So generally they will be close if not the same as `master` (ie the main development branch). But hiccups can happen.

The latest actions on the main branch are always whatever its up to :)

**Comment by discip** [comment, marked as answer]
Do it like this and you will have everything already merged:

1.
![image](https://user-images.githubusercontent.com/53649486/134971705-f28afb6b-c16a-4998-a392-5cee43c2211a.png)

2.
![image](https://user-images.githubusercontent.com/53649486/134971915-b5403378-ab4a-419a-b767-22abbaca5f22.png)

3.
![image](https://user-images.githubusercontent.com/53649486/134972124-4031923e-4b4b-4952-9135-8d5b99151f73.png)

4.
![image](https://user-images.githubusercontent.com/53649486/134972400-2146e373-42ec-4beb-8236-cee6907775c2.png)

kind regards

**Reply by JosephJilek** [reply]
Thanks

**Reply by discip** [reply]
You are welcome. 😊
